sections/Titans stronghold.tex

```diff
@@ -0,0 +1,174 @@
+\end{itemize}
+
+\textbf{(2*P) × Far Map tile (II-III)}
+
+\textbf{(2*P) × Near Map tile (IV-V)}
+
+\textbf{(1*P) + 1 × Center Map tile (VI-VII)}
+
+If you don't have enough VI-VII tile you can use another for tile near to tile I.
+The center of this tile is the Titans' Stronghold.
+
+\columnbreak
+
+\subsection*{\MakeUppercase{Win conditions}}
+
+\textbf{Win:}
+\begin{itemize}
+	\item Kill all unit in the Titans' Stronghold (the center of tile VI-VII next to the tiles I)
+\end{itemize}
+
+\subsection*{\MakeUppercase{Timed events}}
+
+\textbf{4$^{th}$, 8$^{th}$ and 12$^{th}$ Round:}
+\begin{itemize}
+	\item Remove black cubes from windmills, water wheels and mystical gardens.
+\end{itemize}
+
+\subsection*{\MakeUppercase{The story}}
+
+TODO \end{multicols*}
+
+\newpage
+
+\subsection*{\MakeUppercase{Additional rules}}
+
+\begin{itemize}
+	\item Remember : the center of tile VI-VII next to all I, is the Titans' Stronghold.
+	\item nobody can enter the titans' Stronghold before all other VII battle are flagged by any player.
+	\item After defeating a level VII neutral army, instead of resolving field the player chooses an option three times from the following list (an option may be chosen multiple times):
+	\begin{itemize}
+		\item Another player (you choose) gains 5 gold
+		\item Another player (you choose) gains 2 building materials
+		\item Another player (you choose) gains 1 valuable
+	\end{itemize}
+	Then flag the VII with faction cube.
+	(In solo play : no bonus!)
+	\item Ignore all yellow line in tiles I.
+	\item You can use your build token to give your resources to another player.
+	\item Two player can use your build token for exchange artifacts and/or spells.
+	\item Whenever a player visits an Obelisk, that player rolls one tresure dice and one resource dice, chooses one of the dice and resolves its outcome dice.
+	\item When all VII (not includ the titans' Stronghold) are flagged, randomly draw and shuffle the following (next page) number of neutral Unit cards from each of their corresponding decks to create a separate deck of neutral Units for the titans' Stronghold (The deck of titans' Stronghold is sometimes cut in two because otherwise certain scenarios are impossible to implement with the cards of certain extensions.)
+	\item Any time a Hero enters the titans' Stronghold they draw 5 cards from the Titans' Stronghold deck instead of drawing them from the Neutral Unit card decks. The units are placed on the Combat board (see page 29, “Neutral Unit Setup” in the Core Rulebook). Players attempt to beat the units they find at the Titans' Stronghold. Any Neutral Units defeated during Combat in the Titans' Stronghold are returned to their respective Neutral Unit decks instead of Titans' Stronghold deck. Any Neutral Units surviving Combat at the Titans' Stronghold are shuffled back into the Titans' Stronghold deck. If there are not enough Unit cards in this deck, draw as many Unit cards as there are and place them on the Combat board.
```

Comment:
```suggestion
	\item Any time a Hero enters the Titans' Stronghold they draw 5 cards from the Titans' Stronghold deck instead of drawing them from the Neutral Unit card decks. The units are placed on the Combat board (see page 29, “Neutral Unit Setup” in the Core Rulebook). Players attempt to beat the units they find at the Titans' Stronghold. Any Neutral Units defeated during Combat in the Titans' Stronghold are returned to their respective Neutral Unit decks instead of Titans' Stronghold deck. Any Neutral Units surviving Combat at the Titans' Stronghold are shuffled back into the Titans' Stronghold deck. If there are not enough Unit cards in this deck, draw as many Unit cards as there are and place them on the Combat board.
```